### ISSUE TITLE
Create an environment: Failed to parse user token.

### ISSUE THREAD

**Issue by tp6m35p4** [MicrosoftDocs/vscodespaces#257]
I got this message when I was trying to **create new environment** and **register local environment**. 
I had tried re-sign in and re-install the plugin and it still doesn't work.
It happened on **win10/macOS**.

OS: macOS Catalina 10.15 / windows 10
VS code version: 1.41
Subscription: Azure for student.

Here is the log on macOS.
```
[2019-12-16 06:09:11.727 Client I] User logged in as <myaccount>@gmail.com
[2019-12-16 06:09:18.298 Command:l I] Decorator starting: l
[2019-12-16 06:09:18.298 Command:l I] Decorator starting: a
[2019-12-16 06:09:23.690 Command:l E] Command [Create an environment]: Add New Environment Failure: Failed to parse user token. (5391ms)
[2019-12-16 06:09:23.691 Command:l I] Decorator finishing: a. Duration: 659ms (1 + 658)
[2019-12-16 06:09:33.537 Command:d I] Decorator starting: l
[2019-12-16 06:09:33.537 Command:d I] Decorator starting: a
```

Thanks for your help.

**Comment by taiyosogawa** [comment]
@tp6m35p4, thank you for reaching out - we're taking a look into this issue.

I believe this is a duplicate of #96. Could you try creating a plan in [https://online.visualstudio.com/environments](https://online.visualstudio.com/environments)? Hopefully that will unblock you for the moment and help us identify the root cause of the issue.

**Comment by tp6m35p4** [comment]
@taiyosogawa I had tried creating plan on that website, but after I click sign in, it just redirect to the same page.

**Comment by taiyosogawa** [comment]
Sorry for the very late reply. 

When you sign in, there should be a pop-up to sign in like this. Could you check if you have pop-ups blocked? 

![image](https://user-images.githubusercontent.com/1176981/71687665-40262a80-2d53-11ea-84d2-501a20cfda95.png)

**Comment by tp6m35p4** [comment]
@taiyosogawa yes, here is the screen recording.
https://youtu.be/WJCAjL6VtM0

Thank for your reply.

**Comment by taiyosogawa** [comment]
Thanks @tp6m35p4, this is very helpful. @plisy, have you seen this behavior before?

**Comment by tp6m35p4** [comment]
For more information, it disappeared automatically due to I guess I had already login. So I tried to use "Private mode", but after I input my account and password and clicked "NEXT" it still disappear and nothing happened. It happened on Edge, Firefox and Chrome.

**Comment by littleone2018** [comment]
@taiyosogawa
http-client.ts:30 Cross-Origin Read Blocking (CORB) blocked cross-origin response https://vortex.data.microsoft.com/collect/v1 with MIME type application/json. See https://www.chromestatus.com/feature/5629709824032768 for more details.

**Comment by lacafjh** [comment]
I got the same message tonight. I turned into Chrome but the pop-up just disappeared when I tried to sign in. I hope that can be resolved.

**Comment by taiyosogawa** [comment]
@lacafjh, I'm sorry - that can be very frustrating. Can you check if #184 looks like the issue you're seeing?

**Comment by lacafjh** [comment]
@taiyosogawa I succeeded later in New Microsoft Edge. The pop-up stayed for a few seconds, just allowing me to finish the process. Thanks for your reply.

**Comment by lostintangent** [comment]
Closing this as resolved per the previous comment. Please reopen this if you’re still running into a problem. Thanks!
